samples/drivers/espi/prj_mec1501modular_assy6885.conf

```diff
@@ -7,3 +7,7 @@ CONFIG_LOG_PROCESS_THREAD_SLEEP_MS=100
 CONFIG_ESPI_AUTOMATIC_WARNING_ACKNOWLEDGE=n
 # Sample code doesn't handle ACPI host communication
 CONFIG_ESPI_PERIPHERAL_HOST_IO=n
+# Uncomment to test SAF EC portal API's
+#CONFIG_ESPI_SAF=y
```

Comment:
ditto

Reply:
@albertofloyd recommended SAF tests be enabled only for the evaluation board (mec15xxevb_assy6853).
Removed commented out SAF config items from the prj.conf and prj_mec1501modular_assy6885.conf.
Uncommented SAF config enables in prj_mec15xxevb_assy6853.conf
Updated the SAF driver Kconfig to be disabled by default.